.github/workflows/update-web.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will automatically create a PR in deephaven-core to update the WEB_VERSION
+name: Update web version
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types: [update_web]
```

Comment:
So, GitHub says 

> You can use the GitHub API to trigger a webhook event called [repository_dispatch](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#repository_dispatch) when you want to trigger a workflow for activity that happens outside of GitHub

>  You can manually trigger a workflow run using the GitHub API, GitHub CLI, or GitHub browser interface.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

I've not used repository_dispatch before, but I'm wondering if it's still preferable to use "workflow_dispatch" via the API b/c of the wording "for activity that happens outside of GitHub".

Reply:
Good call. I was going based off this post: https://github.com/orgs/community/discussions/26323#discussioncomment-3251443
But looks like it was recently updated to just use workflow_dispatch: https://github.com/orgs/community/discussions/26323#discussioncomment-3251448
I'll just remove the `repository_dispatch` trigger. Triggering the `workflow_dispatch` will still be done from the web-client-ui repo.